Update installation instructions for libmamba (#1519)
TomographicImaging/CIL: README.md

# Installation of CIL

> Binary installation of CIL can be achieved with conda or mamba. mamba's environment solver provides faster environment resolution than conda's so we recommend this route, although in many cases the default conda will still work but may be very slow.
> 
> `miniconda` is a minimal installer for `conda`. Installation instructions can be found [here](https://docs.conda.io/projects/miniconda/en/latest/).
> You can then force your `conda` installation to use `libmamba` instructions are [here](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community)
> 
> Alternatively, `mamba` can be installed via `https://github.com/conda-forge/miniforge` which is a minimal installer for `conda` with support for `mamba`.

Author: ```suggestion
Binary installation of CIL can be achieved with `conda` or `mamba`. `mamba`'s environment solver (`libmamba`) provides faster environment resolution than conda's so we recommend this route, although in many cases the default conda will still work but may be very slow.

`miniconda` is a minimal installer for `conda`. Installation instructions can be found [here](https://docs.conda.io/projects/miniconda/en/latest/).
You can then force your `conda` installation to use `libmamba` instructions are [here](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community).

Alternatively, `mamba` can be installed via `https://github.com/conda-forge/miniforge` which is another minimal installer for `conda` with optional support for `mamba`. In this case replace `conda` with `mamba` in the below commands.
```

Install a new environment using:

```bash

> conda/mamba create --name cil -c conda-forge -c intel -c ccpi cil=23.1.0

Reviewer: I think `conda/mamba` is too confusing and some people will literally just type that. Perhaps just write 'mamba' and below explain to substitute 'mamba' with 'conda' if using 'conda'

```

To install CIL and the additional packages and plugins needed to run the [CIL demos](https://github.com/TomographicImaging/CIL-Demos) install the environment with:

```bash

> conda/mamba create --name cil -c conda-forge -c intel -c ccpi cil=23.1.0 astra-toolbox tigre ccpi-regulariser tomophantom "ipywidgets<8"

Reviewer: again conda/mamba

> conda/mamba create --name cil -c conda-forge -c intel -c ccpi cil=23.1.0 astra-toolbox tigre ccpi-regulariser tomophantom "ipywidgets<8"

Author: ```suggestion
conda create --name cil -c conda-forge -c intel -c ccpi cil=23.1.0 astra-toolbox tigre ccpi-regulariser tomophantom "ipywidgets<8"
```

# Installation of CIL

Binary installation of CIL can be achieved with `conda` or `mamba`. `mamba`'s environment solver (`libmamba`) provides faster environment resolution so we recommend this route, although in many cases the default `conda` will still work but may be very slow.

`miniconda` is a minimal installer for `conda`. Installation instructions can be found [here](https://docs.conda.io/projects/miniconda/en/latest/).

> You can then force your `conda` installation to use `libmamba` instructions are [here](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community).

Reviewer: ```suggestion
You can then force your `conda` installation to use `libmamba`  - instructions are [here](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community).
```
